doc/usage/extensions/coverage.rst

```diff
@@ -6,15 +6,58 @@
 
 This extension features one additional builder, the :class:`CoverageBuilder`.
 
+.. todo:: Write this section.
```

Comment:
Will this TODO remain?

Reply:
Not long-term. I have a separate series but GitHub doesn't let me stack PRs. I'll push once this is merged.